src/lib/helpers/config.helper.ts

```diff
@@ -16,51 +13,12 @@ export const defaultUploadOptions: UploadOptions = {
   ipfsGateway: defaultIpfsGateway,
 };
 
-export const LSP3_UP_KEYS = {
-  UNIVERSAL_RECEIVER_DELEGATE_KEY: keccak256(toUtf8Bytes('LSP1UniversalReceiverDelegate')),
-  LSP3_PROFILE: keccak256(toUtf8Bytes('LSP3Profile')),
-};
-
-export const LSP4_KEYS = {
-  LSP4_METADATA: keccak256(toUtf8Bytes('LSP4Metadata')),
-  LSP4_CREATORS_ARRAY: keccak256(toUtf8Bytes('LSP4Creators[]')),
-  LSP4_CREATORS_MAP_PREFIX: '0x6de85eaf5d982b4e5da00000',
-};
-
-export const NULL_ADDRESS = '0x0000000000000000000000000000000000000000';
-
-export const ADDRESS_PERMISSIONS_ARRAY_KEY =
-  '0xdf30dba06db6a30e65354d9a64c609861f089545ca58c6b4dbe31a5f338cb0e3';
-
-export const PREFIX_PERMISSIONS = '0x4b80742de2bf82acb3630000';
+export const DEFAULT_CONTRACT_VERSION = lspSmartContractsVersion;
 
-// exclude DELEGATECALL for safety
-export const DEFAULT_PERMISSIONS: Permissions = {
-  CHANGEOWNER: true,
-  EDITPERMISSIONS: true,
-  ADDCONTROLLER: true,
-  SETDATA: true,
-  CALL: true,
-  STATICCALL: true,
-  DELEGATECALL: false,
-  DEPLOY: true,
-  TRANSFERVALUE: true,
-  SIGN: true,
-  SUPER_CALL: true,
-  SUPER_DELEGATECALL: false,
-  SUPER_SETDATA: true,
-  SUPER_STATICCALL: true,
-  SUPER_TRANSFERVALUE: true,
-  ENCRYPT: true,
-  ADDEXTENSIONS: true,
-  ADDUNIVERSALRECEIVERDELEGATE: true,
-  CHANGEEXTENSIONS: true,
-  CHANGEUNIVERSALRECEIVERDELEGATE: true,
-  DECRYPT: true,
-  REENTRANCY: true,
+// TODO: add this constant inside the `@lukso/lsp-smart-contracts` package
+export const JSONURL_KNOWN_HASH_FUNCTIONS = {
+  'keccak256(utf8)': '0x6f357c6a',
 };
```

Comment:
What is the constant representing exactly ? Thanks

Reply:
@magalimorin18 The type of hash function used to generate a JSONURL.

See this in LSP2 specs: https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md#jsonurl

![image](https://github.com/lukso-network/tools-lsp-factory/assets/31145285/7445aac1-4f8b-4d7c-bd60-630d04c1b5db)

Reply:
@magalimorin18 I haver removed this constant to import it directly from _erc725.js_. So that we don't have to define it in multiple places. It is related to LSP2 and _erc725.js_ directly anyway.

https://github.com/lukso-network/tools-lsp-factory/pull/226/commits/03b50dc9a6a1cbf986ca9d0d3f6a1dcb0f0db26e